specs/swagger.yml

```diff
@@ -4802,3 +5051,80 @@ definitions:
 
   specBodyDetail:
     type: string
+
+  listRoleAttributesEnvelope:
+    type: object
+    properties:
+      meta:
+        $ref: '#/definitions/meta'
+      data:
+        $ref: '#/definitions/roleAttributesList'
+
+  roleAttributesList:
+    description: An array of role attributes
+    type: array
+    items:
+      type: string
+
+  getIdentityPolicyAdviceEnvelope:
+    type: object
+    properties:
+      meta:
+        $ref: '#/definitions/meta'
+      data:
+        $ref: '#/definitions/policyAdvice'
+
+  policyAdvice:
+    type: "object"
+    required:
+      - "identityId"
+      - "identity"
+      - "serviceId"
+      - "service"
+      - "isBindAllowed"
+      - "isDialAllowed"
+      - "identityRouterCount"
+      - "serviceRouterCount"
+      - "commonRouters"
+    properties:
+      identityId:
+        type: "string"
```

Comment:
With `id` values we should just go all string.

If you are in this file again I would just find: `format: uuid` and remove them all. We have to be 100% consistent otherwise consumers of this will need to convert string <-> uuid.

Reply:
Done. I remove `format: uuid` everywhere it was used by ids, but I left other uses, such as for tokens, etc